src/main/java/oss/fosslight/util/ExcelUtil.java

```diff
@@ -73,7 +73,24 @@
 	// Mapper
 	private static ProjectMapper 	projectMapper 		= (ProjectMapper) 		getWebappContext().getBean(ProjectMapper.class);
 	private static CodeMapper 		codeMapper 			= (CodeMapper) 			getWebappContext().getBean(CodeMapper.class);
-	
+
+	// Sheet names that starts with specific word
+	public static List<String> getSheetNameStartsWith(String word, List<UploadFile> list, String excelLocalPath) throws InvalidFormatException, IOException {

```

Comment:
It returns Sheet no, not Sheet name, so how about change the function name to "getSheetNoStartsWith"?

Reply:
@soimkim 
I agree, it will be clearer 👍